Docs for some errors (#144)
hyperledger/iroha: docs/source/api/queries.rst

    :header: "Code", "Error Name", "Description", "How to solve"

    "1", "Could not get block", "Internal error happened", "Try again or contact developers"
    "2", "No such permissions", "Query's creator does not have a permission to get block", "Grant `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission"
    "3", "Invalid signatures", "Signatures of this query did not pass validation", "Add more signatures and make sure query's signatures are a subset of account's signatories"
    "3", "Invalid height", "Supplied height is not uint_64 or greater than the ledger's height", "Check the height and try again"

.. note::

>     Error code 3 is ambiguos for this method.

Reviewer: ```suggestion
    Error code 3 is ambiguous for this query.
```

>     It indicates either invalid signatories or invalid height.

Reviewer: The same error code should not be used for two kinds of issue.
This most likely is a bug.

Reviewer: Just for historical purposes. A related issue is https://jira.hyperledger.org/browse/IR-589

    :header: "Code", "Error Name", "Description", "How to solve"

    "1", "Could not get block streaming", "Internal error happened", "Try again or contact developers"
    "2", "No such permissions", "Query's creator does not have any of the permissions to get blocks", "Grant `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission"
    "3", "Invalid signatures", "Signatures of this query did not pass validation", "Add more signatures and make sure query's signatures are a subset of account's signatories"

.. note::
    `BlockErrorResponse` contains only `message` field.

>     In case of stateful validation error it will be `"stateful invalid"`.

Reviewer: ```suggestion
    In case of stateful validation error it will be "stateful invalid".
```

      }
    }

>     message BlockErrorResponse {

Reviewer 2: Maybe also add `BlockResponse` before `BlockErrorResponse`?

    :header: "Code", "Error Name", "Description", "How to solve"

    "1", "Could not get block streaming", "Internal error happened", "Try again or contact developers"
    "2", "No such permissions", "Query's creator does not have any of the permissions to get blocks", "Grant `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission"
    "3", "Invalid signatures", "Signatures of this query did not pass validation", "Add more signatures and make sure query's signatures are a subset of account's signatories"

.. note::
    `BlockErrorResponse` contains only `message` field.
    In case of stateful validation error it will be "stateful invalid".
    `GetBlock <#get-block>`__ requires same `can_get_block <../maintenance/permissions.html#can-get-blocks>`__ permission.

>     Therefore, it can be used with `height = 1` (first block is always present) to get verbose error.

Reviewer 3: Not clear enough for me. What do you mean by *verbose error*? If it is about checking one's permissions - please write that, otherwise please explain the case you mean.

>     Therefore, it can be used with `height = 1` (first block is always present) to check for invalid signatories are insufficient permissions.

Reviewer 2: ```suggestion
    Therefore, it can be used with `height = 1` (first block is always present) to check for invalid signatories or insufficient permissions.
```
If I understood the idea correctly
